Use charge's currency in email template

johnfraney opened the pull request:
Display the charge's actual currency rather than `USD` in the email body

johnfraney committed:
Use charge's currency in email template

Display the charge's actual currency rather than `USD` in the email body

kavdev commented on djstripe/templates/djstripe/email/body_base.txt:
```diff
@@ -13,7 +13,7 @@ DETAILS
 {{ charge.customer.subscription.plan.name }} - ${{ charge.amount|floatformat:2 }}
 
-TOTAL: ${{ charge.amount|floatformat:2 }} USD
+TOTAL: ${{ charge.amount|floatformat:2 }} {{ charge.currency|upper }}
```
Any preference on the "$"? I'd say removing it might be best.

johnfraney replied:
@kavdev That's a good point! Removing it would be better for non-dollar-based currencies. It made sense for me here in Canada, but it's a big world out there.

johnfraney committed:
Update body_base.txt

Remove code from receipt email that assumes the user has paid with a dollar-based currency.

kavdev reviewed (changes requested):
Looks great! Can you also please tackle https://github.com/kavdev/dj-stripe/blob/1.0.0/djstripe/templates/djstripe/includes/_history_table.html while you're at it?

Also, there's a test that needs fixing.
